### PR TITLE
fix getFileMetadata and getFileLeaf that the param will be @NonNull

### LINKED ISSUE #8207: Change getFileMetadata and getFileLeaf that the param will be @NonNull
Change getFileMetadata and getFileLeaf that the param will be @NonNull

### PR DESCRIPTION
**Pull request by povolev15**
fix getFileMetadata and getFileLeaf that the param will be @NonNull

**Related issue(s)**:

Fixes #8207

### CODE REVIEW

**Review by Nana-EC — DISMISSED** [review]

**Review by lukelee-sl — APPROVED** [review]
LGTM

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java [hunk 1]
```diff
@@ -77,9 +77,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final var transactionBody = context.body().fileAppendOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, false);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java:81]: These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileDeleteHandler.java [hunk 2]
```diff
@@ -65,9 +65,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final var transactionBody = context.body().fileDeleteOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, true);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileDeleteHandler.java:69]: These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java [hunk 3]
```diff
@@ -151,6 +151,7 @@ private Optional<FileInfo> infoForFile(
                         file.deleted());
             }
         } else {
+            requireNonNull(fileID);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java:154]: This is unnecessary, you cannot reach this line if fileID is null.

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemDeleteHandler.java [hunk 4]
```diff
@@ -63,9 +63,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final var transactionBody = context.body().systemDeleteOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, true);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemDeleteHandler.java:67]: These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemUndeleteHandler.java [hunk 5]
```diff
@@ -62,9 +62,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         final var transactionBody = context.body().systemUndeleteOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, true);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemUndeleteHandler.java:66]: These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java [hunk 6]
```diff
@@ -80,9 +80,11 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         requireNonNull(context);
         final var transactionBody = context.body().fileUpdateOrThrow();
         final var fileStore = context.createStore(ReadableFileStore.class);
-        preValidate(transactionBody.fileID(), fileStore, context, false);
+        final var transactionFileId = transactionBody.fileID();
+        requireNonNull(transactionFileId);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java:84]: These lines can be combined.
```suggestion
        final var transactionFileId = requireNonNull(transactionBody.fileID());
```

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java [hunk 7]
```diff
@@ -105,6 +105,7 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
      */
     private @Nullable Optional<FileContents> contentFile(
             @NonNull final FileID fileID, @NonNull final ReadableFileStore fileStore) {
+        requireNonNull(fileID);
```
Comment (jsync-swirlds) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java:108]: This method is private, so this line is unnecessary.  It won't be called with a null fileID.

**Review by jsync-swirlds — COMMENTED** [review]
Some recommended simplifications.

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java (hunk 1 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java:81]: fixed thanks

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileDeleteHandler.java (hunk 2 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileDeleteHandler.java:69]: fixed thanks

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemDeleteHandler.java (hunk 4 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemDeleteHandler.java:67]: fixed thanks

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemUndeleteHandler.java (hunk 5 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileSystemUndeleteHandler.java:66]: fixed thanks

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java (hunk 6 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java:84]: fixed thanks

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java (hunk 7 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java:108]: fixed thanks

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java (hunk 3 above)
Comment (povolev15) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java:154]: fixed thanks

**Review by jsync-swirlds — APPROVED** [review]

**Review by iwsimon — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**